src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,530 @@
+---
+title: Migrate and Replicate Data with Oracle GoldenGate
+summary: Use GoldenGate to migrate data to a CockroachDB cluster.
+toc: true
+docs_area: migrate
+---
+
+[Oracle GoldenGate](https://www.oracle.com/integration/goldengate/) is a managed service that can collect, replicate, and manage transactional data between databases. GoldenGate can use CockroachDB as a sink by leveraging CockroachDB's PostgreSQL compatibility. This page describes how to:
+
+- [Configure Oracle GoldenGate for PostgreSQL](#configure-oracle-goldengate-for-postgresql).
+- [Set up Extract to capture data from a source database](#set-up-extract-to-capture-data-from-a-source-database).
+- [Set up Replicat to deliver data to CockroachDB](#set-up-replicat-to-deliver-data-to-cockroachdb).
+- [Perform bulk replication](#perform-bulk-replication).
+
+As of this writing, GoldenGate supports the following database [sources](https://docs.oracle.com/en/middleware/goldengate/core/21.3/coredoc/configure-databases.html):
```

Comment:
cc @nickvigilante This is an instance where removing `/en/` actually results in a 404.

Reply:
It seems like the exceptions is not functioning as I expect it to. I think this was related to a problem in Vale, not so much my code. I'll investigate again.